src/web-extension/options/components/template-input.jsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import TextareaAutosize from 'react-textarea-autosize';
+
+function TemplateInputElement({ multiline, ...props }) {
+  if (multiline) {
+    return <TextareaAutosize {...props} />;
+  }
+
+  return <input type="text" {...props} />;
+}
+
+TemplateInputElement.propTypes = {
+  multiline: PropTypes.bool.isRequired,
+};
```

Comment:
maybe we can put this into it's own file? Just an idea. It seems also not too terrible here though, so feel free to leave it.